run-cgprna/run_cgprna/map.py

```diff
@@ -72,46 +69,31 @@ def map_seq_files(args):
         reference_data_root=os.path.join(temp_dir, ref_root_dir_name)
         
         # set ref related values for star_mapping.pl
-        for key,value in ref_related_args.items():
-            # GTF file name will be the same as in the bundle
-            if key!= '--gene-build-gtf-name':
-                if value is None:
-                    ref_related_args[key] = ref_related_defaults[key]
-            else:
-                if value is not None:
-                    print('Warning: provided "--gene-build-gtf-name" will be overwritten by the GTF file name in the reference bundle.')
+        for arg_name in REF_RELATED_DEFAULTS.keys():
+            if args_dict[arg_name] is None:
+                print('Set "%s" to default.' % arg_name)
+                args_dict[arg_name] = REF_RELATED_DEFAULTS[arg_name]
+            if arg_name == 'gene_build':
+                print('Make sure a folder named "%s" exists in the ref bundle.' % args_dict[arg_name])
+            if arg_name == 'gene_build_gtf_name':
```

Comment:
An `elif` here might save a couple nanoseconds.

Reply:
I want to both `print` run if arg_name is in [gene_build, gene_build_gtf_name] :)

Reply:
My observation was that in the same iteration `arg_name` can't be both.

Reply:
Ah, I see what you mean.